decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
```

Comment:
- [ OPTIONAL ] I recommend adding here attr-accessor to the namable variable so kindly add it here it will be something like this: 👍🏻

  > attr_accessor :nameable